roachtest: psycopg failed

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.psycopg [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/18900490?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/18900490?buildTab=artifacts#/psycopg) on master @ [c471a724f99a5add3ccefe07995251760d0e2212](https://github.com/cockroachdb/cockroach/commits/c471a724f99a5add3ccefe07995251760d0e2212):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-c471a724f99a5add3ccefe07995251760d0e2212
Tests run against psycopg 3c58e96e1000ef60060fb8139687028cb274838d
488 Total Tests Run
467 tests passed
21 tests failed
245 tests skipped
3 tests ignored
0 tests passed unexpectedly
21 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_returning (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_empty (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_tuples (unexpected)
--- FAIL: tests.test_sql.SqlFormatTests.test_executemany (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_dicts (unexpected)
--- FAIL: tests.test_cursor.NamedCursorTests.test_invalid_name (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_percent_escape (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_unicode (unexpected)
--- FAIL: tests.test_connection.SignalTestCase.test_bug_551_returning (unexpected)
--- FAIL: tests.test_cursor.NamedCursorTests.test_not_scrollable (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_unicode (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_empty (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_tuples (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_composed (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_one (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_many (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_invalid_sql (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_pages (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_one (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_pages (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_many (unexpected)
For a full summary look at the psycopg artifacts 
An updated blocklist (psycopgBlockList) is available in the artifacts' psycopg log
test artifacts and logs in: /artifacts/psycopg/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=azure</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>metamorphicBufferedSender=true</code>
 - <code>metamorphicLeases=leader</code>
 - <code>runtimeAssertionsBuild=true</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

_Grafana is not yet available for azure clusters_</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*psycopg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-48046

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.psycopg [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/18906414?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/18906414?buildTab=artifacts#/psycopg) on master @ [fa8342d50c1bdfbc307f4bca9bc0d34da4475258](https://github.com/cockroachdb/cockroach/commits/fa8342d50c1bdfbc307f4bca9bc0d34da4475258):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-fa8342d50c1bdfbc307f4bca9bc0d34da4475258
Tests run against psycopg 3c58e96e1000ef60060fb8139687028cb274838d
488 Total Tests Run
467 tests passed
21 tests failed
245 tests skipped
3 tests ignored
0 tests passed unexpectedly
21 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: tests.test_cursor.NamedCursorTests.test_not_scrollable (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_tuples (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_invalid_sql (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_one (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_returning (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_pages (unexpected)
--- FAIL: tests.test_cursor.NamedCursorTests.test_invalid_name (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_empty (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_many (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_percent_escape (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_unicode (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_one (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_pages (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_tuples (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_dicts (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_many (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_composed (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_empty (unexpected)
--- FAIL: tests.test_sql.SqlFormatTests.test_executemany (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_unicode (unexpected)
--- FAIL: tests.test_connection.SignalTestCase.test_bug_551_returning (unexpected)
For a full summary look at the psycopg artifacts 
An updated blocklist (psycopgBlockList) is available in the artifacts' psycopg log
test artifacts and logs in: /artifacts/psycopg/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=azure</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicBufferedSender=true</code>
 - <code>metamorphicLeases=epoch</code>
 - <code>runtimeAssertionsBuild=true</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

_Grafana is not yet available for azure clusters_</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*psycopg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.psycopg [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/18912543?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/18912543?buildTab=artifacts#/psycopg) on master @ [cbb292c487ea7413772a5c3c349f93467264c190](https://github.com/cockroachdb/cockroach/commits/cbb292c487ea7413772a5c3c349f93467264c190):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-cbb292c487ea7413772a5c3c349f93467264c190
Tests run against psycopg 3c58e96e1000ef60060fb8139687028cb274838d
488 Total Tests Run
467 tests passed
21 tests failed
245 tests skipped
3 tests ignored
0 tests passed unexpectedly
21 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_returning (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_unicode (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_unicode (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_dicts (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_pages (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_percent_escape (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_empty (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_pages (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_many (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_empty (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_one (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_composed (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_many (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_tuples (unexpected)
--- FAIL: tests.test_cursor.NamedCursorTests.test_invalid_name (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_tuples (unexpected)
--- FAIL: tests.test_cursor.NamedCursorTests.test_not_scrollable (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_invalid_sql (unexpected)
--- FAIL: tests.test_connection.SignalTestCase.test_bug_551_returning (unexpected)
--- FAIL: tests.test_sql.SqlFormatTests.test_executemany (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_one (unexpected)
For a full summary look at the psycopg artifacts 
An updated blocklist (psycopgBlockList) is available in the artifacts' psycopg log
test artifacts and logs in: /artifacts/psycopg/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=azure</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicBufferedSender=true</code>
 - <code>metamorphicLeases=expiration</code>
 - <code>runtimeAssertionsBuild=true</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

_Grafana is not yet available for azure clusters_</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*psycopg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.psycopg [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18912532?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18912532?buildTab=artifacts#/psycopg) on master @ [cbb292c487ea7413772a5c3c349f93467264c190](https://github.com/cockroachdb/cockroach/commits/cbb292c487ea7413772a5c3c349f93467264c190):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-cbb292c487ea7413772a5c3c349f93467264c190
Tests run against psycopg 3c58e96e1000ef60060fb8139687028cb274838d
488 Total Tests Run
467 tests passed
21 tests failed
245 tests skipped
3 tests ignored
0 tests passed unexpectedly
21 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: tests.test_cursor.NamedCursorTests.test_not_scrollable (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_tuples (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_invalid_sql (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_one (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_tuples (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_empty (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_returning (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_empty (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_one (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_pages (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_unicode (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_many (unexpected)
--- FAIL: tests.test_cursor.NamedCursorTests.test_invalid_name (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_many (unexpected)
--- FAIL: tests.test_sql.SqlFormatTests.test_executemany (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_composed (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_percent_escape (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_dicts (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_unicode (unexpected)
--- FAIL: tests.test_connection.SignalTestCase.test_bug_551_returning (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_pages (unexpected)
For a full summary look at the psycopg artifacts 
An updated blocklist (psycopgBlockList) is available in the artifacts' psycopg log
test artifacts and logs in: /artifacts/psycopg/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicBufferedSender=true</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=true</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18912532/psycopg/1740398107092/1740398310051)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*psycopg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.psycopg [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/18923269?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/18923269?buildTab=artifacts#/psycopg) on master @ [8bd38972db32f5e683ba6dc0eb599334919919e5](https://github.com/cockroachdb/cockroach/commits/8bd38972db32f5e683ba6dc0eb599334919919e5):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-8bd38972db32f5e683ba6dc0eb599334919919e5
Tests run against psycopg 3c58e96e1000ef60060fb8139687028cb274838d
488 Total Tests Run
467 tests passed
21 tests failed
245 tests skipped
3 tests ignored
0 tests passed unexpectedly
21 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_invalid_sql (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_tuples (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_percent_escape (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_dicts (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_pages (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_returning (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_empty (unexpected)
--- FAIL: tests.test_sql.SqlFormatTests.test_executemany (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_empty (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_composed (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_one (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_pages (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_many (unexpected)
--- FAIL: tests.test_connection.SignalTestCase.test_bug_551_returning (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_unicode (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_one (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_unicode (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_tuples (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_many (unexpected)
--- FAIL: tests.test_cursor.NamedCursorTests.test_invalid_name (unexpected)
--- FAIL: tests.test_cursor.NamedCursorTests.test_not_scrollable (unexpected)
For a full summary look at the psycopg artifacts 
An updated blocklist (psycopgBlockList) is available in the artifacts' psycopg log
test artifacts and logs in: /artifacts/psycopg/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=azure</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicBufferedSender=true</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=true</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

_Grafana is not yet available for azure clusters_</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*psycopg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.psycopg [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18923258?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18923258?buildTab=artifacts#/psycopg) on master @ [8bd38972db32f5e683ba6dc0eb599334919919e5](https://github.com/cockroachdb/cockroach/commits/8bd38972db32f5e683ba6dc0eb599334919919e5):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-8bd38972db32f5e683ba6dc0eb599334919919e5
Tests run against psycopg 3c58e96e1000ef60060fb8139687028cb274838d
488 Total Tests Run
467 tests passed
21 tests failed
245 tests skipped
3 tests ignored
0 tests passed unexpectedly
21 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_unicode (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_pages (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_invalid_sql (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_many (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_returning (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_pages (unexpected)
--- FAIL: tests.test_cursor.NamedCursorTests.test_not_scrollable (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_many (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_composed (unexpected)
--- FAIL: tests.test_sql.SqlFormatTests.test_executemany (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_empty (unexpected)
--- FAIL: tests.test_connection.SignalTestCase.test_bug_551_returning (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_one (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_empty (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_tuples (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_unicode (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_one (unexpected)
--- FAIL: tests.test_cursor.NamedCursorTests.test_invalid_name (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_percent_escape (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_dicts (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_tuples (unexpected)
For a full summary look at the psycopg artifacts 
An updated blocklist (psycopgBlockList) is available in the artifacts' psycopg log
test artifacts and logs in: /artifacts/psycopg/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicBufferedSender=true</code>
 - <code>metamorphicLeases=expiration</code>
 - <code>runtimeAssertionsBuild=false</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18923258/psycopg/1740489158850/1740489358239)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*psycopg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.psycopg [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/18934383?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/18934383?buildTab=artifacts#/psycopg) on master @ [c44b3134d23d6037e1d5d5e12461a6914aa588e3](https://github.com/cockroachdb/cockroach/commits/c44b3134d23d6037e1d5d5e12461a6914aa588e3):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-c44b3134d23d6037e1d5d5e12461a6914aa588e3
Tests run against psycopg 3c58e96e1000ef60060fb8139687028cb274838d
488 Total Tests Run
467 tests passed
21 tests failed
245 tests skipped
3 tests ignored
0 tests passed unexpectedly
21 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_many (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_empty (unexpected)
--- FAIL: tests.test_sql.SqlFormatTests.test_executemany (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_one (unexpected)
--- FAIL: tests.test_cursor.NamedCursorTests.test_invalid_name (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_many (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_empty (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_pages (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_tuples (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_unicode (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_pages (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_unicode (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_returning (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_composed (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_tuples (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_one (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_invalid_sql (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_percent_escape (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_dicts (unexpected)
--- FAIL: tests.test_cursor.NamedCursorTests.test_not_scrollable (unexpected)
--- FAIL: tests.test_connection.SignalTestCase.test_bug_551_returning (unexpected)
For a full summary look at the psycopg artifacts 
An updated blocklist (psycopgBlockList) is available in the artifacts' psycopg log
test artifacts and logs in: /artifacts/psycopg/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=azure</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicBufferedSender=true</code>
 - <code>metamorphicLeases=epoch</code>
 - <code>runtimeAssertionsBuild=true</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

_Grafana is not yet available for azure clusters_</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*psycopg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.psycopg [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18934372?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18934372?buildTab=artifacts#/psycopg) on master @ [c44b3134d23d6037e1d5d5e12461a6914aa588e3](https://github.com/cockroachdb/cockroach/commits/c44b3134d23d6037e1d5d5e12461a6914aa588e3):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-c44b3134d23d6037e1d5d5e12461a6914aa588e3
Tests run against psycopg 3c58e96e1000ef60060fb8139687028cb274838d
488 Total Tests Run
467 tests passed
21 tests failed
245 tests skipped
3 tests ignored
0 tests passed unexpectedly
21 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_percent_escape (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_empty (unexpected)
--- FAIL: tests.test_connection.SignalTestCase.test_bug_551_returning (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_empty (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_tuples (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_unicode (unexpected)
--- FAIL: tests.test_cursor.NamedCursorTests.test_not_scrollable (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_one (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_pages (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_returning (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_composed (unexpected)
--- FAIL: tests.test_cursor.NamedCursorTests.test_invalid_name (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_invalid_sql (unexpected)
--- FAIL: tests.test_sql.SqlFormatTests.test_executemany (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_many (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_many (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_tuples (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_unicode (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_pages (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_one (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_dicts (unexpected)
For a full summary look at the psycopg artifacts 
An updated blocklist (psycopgBlockList) is available in the artifacts' psycopg log
test artifacts and logs in: /artifacts/psycopg/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>metamorphicBufferedSender=false</code>
 - <code>metamorphicLeases=expiration</code>
 - <code>runtimeAssertionsBuild=true</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18934372/psycopg/1740575040270/1740575241192)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*psycopg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.psycopg [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/18944252?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/18944252?buildTab=artifacts#/psycopg) on master @ [32fe7ee0b575276bff803469006bba08737840cf](https://github.com/cockroachdb/cockroach/commits/32fe7ee0b575276bff803469006bba08737840cf):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-32fe7ee0b575276bff803469006bba08737840cf
Tests run against psycopg 3c58e96e1000ef60060fb8139687028cb274838d
488 Total Tests Run
467 tests passed
21 tests failed
245 tests skipped
3 tests ignored
0 tests passed unexpectedly
21 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_percent_escape (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_unicode (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_pages (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_tuples (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_one (unexpected)
--- FAIL: tests.test_sql.SqlFormatTests.test_executemany (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_many (unexpected)
--- FAIL: tests.test_cursor.NamedCursorTests.test_not_scrollable (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_invalid_sql (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_composed (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_dicts (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_empty (unexpected)
--- FAIL: tests.test_cursor.NamedCursorTests.test_invalid_name (unexpected)
--- FAIL: tests.test_connection.SignalTestCase.test_bug_551_returning (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_many (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_returning (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_tuples (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_unicode (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_empty (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_one (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_pages (unexpected)
For a full summary look at the psycopg artifacts 
An updated blocklist (psycopgBlockList) is available in the artifacts' psycopg log
test artifacts and logs in: /artifacts/psycopg/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=azure</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>metamorphicBufferedSender=true</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=true</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

_Grafana is not yet available for azure clusters_</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*psycopg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.psycopg [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18944241?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18944241?buildTab=artifacts#/psycopg) on master @ [32fe7ee0b575276bff803469006bba08737840cf](https://github.com/cockroachdb/cockroach/commits/32fe7ee0b575276bff803469006bba08737840cf):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-32fe7ee0b575276bff803469006bba08737840cf
Tests run against psycopg 3c58e96e1000ef60060fb8139687028cb274838d
488 Total Tests Run
467 tests passed
21 tests failed
245 tests skipped
3 tests ignored
0 tests passed unexpectedly
21 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_percent_escape (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_composed (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_one (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_tuples (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_pages (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_empty (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_many (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_returning (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_empty (unexpected)
--- FAIL: tests.test_cursor.NamedCursorTests.test_not_scrollable (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_invalid_sql (unexpected)
--- FAIL: tests.test_cursor.NamedCursorTests.test_invalid_name (unexpected)
--- FAIL: tests.test_connection.SignalTestCase.test_bug_551_returning (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_unicode (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_pages (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_many (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_one (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_tuples (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteValues.test_dicts (unexpected)
--- FAIL: tests.test_sql.SqlFormatTests.test_executemany (unexpected)
--- FAIL: tests.test_fast_executemany.TestExecuteBatch.test_unicode (unexpected)
For a full summary look at the psycopg artifacts 
An updated blocklist (psycopgBlockList) is available in the artifacts' psycopg log
test artifacts and logs in: /artifacts/psycopg/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicBufferedSender=false</code>
 - <code>metamorphicLeases=leader</code>
 - <code>runtimeAssertionsBuild=true</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18944241/psycopg/1740641777719/1740641980357)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*psycopg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/cmd/roachtest/tests/psycopg_blocklist.go
+++ b/pkg/cmd/roachtest/tests/psycopg_blocklist.go
@@ -16,7 +16,42 @@ package tests
 // Please keep these lists alphabetized for easy diffing.
 // After a failed run, an updated version of this blocklist should be available
 // in the test log.
-var psycopgBlockList = blocklist{}
+var psycopgBlockList = blocklist{
+	// This test fails because "defaultdb.public.test551" already exists. An
+	// earlier test creates the same table. However, our autocommit_before_ddl
+	// behavior commits that transaction, whereas it was expected to roll back
+	// when the test ended.
+	"tests.test_connection.SignalTestCase.test_bug_551_returning": "142047",
+	// These tests fail with "DECLARE CURSOR can only be used in transaction blocks".
+	// This happens because of our autocommit_before_ddl behavior. They all
+	// create a table, which closes the existing transaction, causing the
+	// above error to occur when DECLARE CURSOR is used.
+	"tests.test_cursor.NamedCursorTests.test_invalid_name":   "142047",
+	"tests.test_cursor.NamedCursorTests.test_not_scrollable": "142047",
+	// The `*executemany*` tests fail because they attempt to create a
+	// table that already exists. This is due to our auto-commit behavior before DDL.
+	// Each failing test creates the same table in a transaction, performs the test,
+	// and then rolls back the transaction. However, we end up committing the first
+	// `CREATE TABLE`, causing all subsequent tests that follow the same pattern to fail.
+	"tests.test_fast_executemany.TestExecuteBatch.test_empty":           "142047",
+	"tests.test_fast_executemany.TestExecuteBatch.test_many":            "142047",
+	"tests.test_fast_executemany.TestExecuteBatch.test_one":             "142047",
+	"tests.test_fast_executemany.TestExecuteBatch.test_pages":           "142047",
+	"tests.test_fast_executemany.TestExecuteBatch.test_tuples":          "142047",
+	"tests.test_fast_executemany.TestExecuteBatch.test_unicode":         "142047",
+	"tests.test_fast_executemany.TestExecuteValues.test_composed":       "142047",
+	"tests.test_fast_executemany.TestExecuteValues.test_dicts":          "142047",
+	"tests.test_fast_executemany.TestExecuteValues.test_empty":          "142047",
+	"tests.test_fast_executemany.TestExecuteValues.test_invalid_sql":    "142047",
+	"tests.test_fast_executemany.TestExecuteValues.test_many":           "142047",
+	"tests.test_fast_executemany.TestExecuteValues.test_one":            "142047",
+	"tests.test_fast_executemany.TestExecuteValues.test_pages":          "142047",
+	"tests.test_fast_executemany.TestExecuteValues.test_percent_escape": "142047",
+	"tests.test_fast_executemany.TestExecuteValues.test_returning":      "142047",
+	"tests.test_fast_executemany.TestExecuteValues.test_tuples":         "142047",
+	"tests.test_fast_executemany.TestExecuteValues.test_unicode":        "142047",
+	"tests.test_sql.SqlFormatTests.test_executemany":                    "142047",
+}
 
 var psycopgIgnoreList = blocklist{
 	"tests.test_async.AsyncTests.test_flush_on_write":           "44709",

